### DISCUSSION TITLE
Wire gauge for S21 Connection

### DISCUSSION THREAD

**Discussion by benswinburne** [revk/xESP32-Faikout#344] [Q&A]
Hi,

I bought a Faikin, bought some premade cables and plugged it all in and it works, great!

However, in re reading the instructions just to double check I did everything I noticed in the cable making instructions the cable is 22AWG, but the cables I have are 24AWG.

Is anyone able to confirm whether or not the 24AWG cables will be capable of the current draw of the Faikin? Or able to tell me the current draw or point me in the direction of where I might find the current draw of the Faikin board?

Thanks

**Comment by njh** [comment]
The Faikin current draw is very low. I don't know for sure but I think it would be around 250mA max.
A bit of Googling says that 24 AWG should be able to handle 2 amps.

I have actually been planning to buy some lower thickness (26 AWG?) cable, because I found 24 AWG a bit too fat to easily crimp.
Although the thickness of the insulation was more of an issue.

**Reply by benswinburne** [reply]
Thanks for the reply. 

Would be great to have some of the specifications like this listed in the repo if possible.

I put 12v 0.25a into an AWG calculator and it recommended a minimum of 25AWG so perhaps worth taking into consideration before buying 26AWG.

FWIW or if it's useful to anyone else, these are the 24 AWG cables I bought

- https://www.aliexpress.us/item/3256805279169465.html - 5 pin option
- https://www.aliexpress.us/item/3256805786266894.html - 5 pin same direction options

**Reply by njh** [reply]
Sorry just looking into this in a bit more detail based on the [ESP32-S3-MINI datasheet](https://www.espressif.com/sites/default/files/documentation/esp32-s3-mini-1_mini-1u_datasheet_en.pdf).

Highest power consumption is when transmitting at 1 Mbps, @20.5 dBm:
* 355 mA at 3.3v = 1.1715 watts

Which is 100 mA at 12 volts. But that is peak and I don't imagine that it will be using anything like that much most of the time.
